src/misc.ts

```diff
@@ -92,3 +92,85 @@ export const hasProperty = (
   object: RuntimeObject,
   name: string | number | symbol,
 ): boolean => Object.hasOwnProperty.call(object, name);
+
+export type PlainObject = Record<number | string | symbol, unknown>;
+
+/**
+ * Predefined sizes (in Bytes) of specific parts of JSON structure.
+ */
+export enum JsonSize {
+  NULL = 4,
+  COMMA = 1,
+  WRAPPER = 1,
+  TRUE = 4,
+  FALSE = 5,
+  ZERO = 1,
+  QUOTE = 1,
+  COLON = 1,
+  DOT = 1,
+}
```

Comment:
Can you follow the TypeScript naming conventions for enums?

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Following it now. Changed in the next commit.